README.md

```diff
@@ -775,6 +775,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 ### Audio Streaming
 
   * [Ampache](http://ampache.org/) - A web based audio/video streaming application. ([Demo](http://play.dogmazic.net/), [Source Code](https://github.com/ampache/ampache)) `AGPLv3` `PHP`
+  * [Beets](http://beets.io/) - Music library manager and MusicBrainz tagger. ([Source code](https://github.com/beetbox/beets)) `MIT` `Python`
```

Comment:
Terminal application, therefore not a web app/network service and does not qualify for the list. Very nice tool though.

Reply:
Beets *is* a network service *and* a Web app, though. That's its entire point, to be a self-hosted network-capable music library manager that can be installed in a variety of configurations using its readily-available Free Software plugins to provide Web-based audio playing/browsing or a music streaming server.

Reply:
I didn't know beets had a web interface, but their website reads:

> Browse your music library graphically through a Web browser

I'll reopen this PR but it would be nice to link to a more specific page about the web interface

Reply:
And probably change the description to `Music library manager and MusicBrainz tagger (command-line & web interface).`